src/main/java/gov/usgs/earthquake/distribution/WebSocketNotificationReceiver.java

```diff
@@ -66,7 +65,9 @@ public void configure(Config config) throws Exception {
     attempts = Integer.parseInt(config.getProperty(CONNECT_ATTEMPTS_PROPERTY, DEFAULT_CONNECT_ATTEMPTS));
     timeout = Long.parseLong(config.getProperty(CONNECT_TIMEOUT_PROPERTY, DEFAULT_CONNECT_TIMEOUT));
     retryOnClose = Boolean.parseBoolean(config.getProperty(RETRY_ON_CLOSE_PROPERTY, DEFAULT_RETRY_ON_CLOSE));
-    trackingFileName = config.getProperty(TRACKING_FILE_NAME_PROPERTY, DEFAULT_TRACKING_FILE_NAME);
+
+    String trackingFileName = config.getProperty(TRACKING_FILE_NAME_PROPERTY, DEFAULT_TRACKING_FILE_NAME);
+    trackingFile = new JSONTrackingFile(trackingFileName);
```

Comment:
What if `configure` is never called?  Maybe move the read inside readTrackingFile (see comment below), and return if the tracking file object is still null?

Reply:
Fixed by searching for a tracking file of the default name on startup. We can discuss alternative options as well.